testing/upload_validation.ts

```diff
@@ -1326,6 +1354,17 @@ function validateObjectSchemaDeprecatedFields(
     pathPrefix,
     context,
   });
+
+  if (schema.identity?.attribution) {
```

Comment:
Doesn't seem that critical, but I noticed that the unit tests don't cover this warning

Reply:
Ah I had a feeling I forgot a case, thanks!